src/legacy/core_plugins/telemetry/server/routes/telemetry_config.ts

```diff
@@ -20,20 +20,46 @@
 import Joi from 'joi';
 import { boomify } from 'boom';
 import { CoreSetup } from 'src/core/server';
+import {
+  TelemetrySavedObjectAttributes,
+  updateTelemetrySavedObject,
+} from '../telemetry_repository';
 
 interface RegisterOptInRoutesParams {
   core: CoreSetup;
   currentKibanaVersion: string;
 }
 
-export interface SavedObjectAttributes {
-  enabled?: boolean;
-  lastVersionChecked: string;
-}
-
-export function registerOptInRoutes({ core, currentKibanaVersion }: RegisterOptInRoutesParams) {
+export function registerTelemetryConfigRoutes({
+  core,
+  currentKibanaVersion,
+}: RegisterOptInRoutesParams) {
   const { server } = core.http as any;
 
+  server.route({
+    method: 'POST',
+    path: '/api/telemetry/v2/usageFetcher',
+    options: {
+      validate: {
+        payload: Joi.object({
+          usageFetcher: Joi.string()
+            .allow(['browser', 'server'])
+            .required(),
```

Comment:
Sorry to labour the point ( I try not to repeat myself in CRs, as I know it's annoying ;)) but this is another example of why using Enums could be useful - we repeat the string values here, and dropping all possible values into a single object could let us reuse the values here without needing to repeat definitions (which adds cognitive load).

Done, won't mention it again. ;)